### ISSUE TITLE
Build error on android and not working host mode on android.

### ISSUE THREAD

**Issue by ymkim50** [FirstGearGames/FishNet#85]
- Version: 2.3.10 Pro

1. "Build And Run" error on android
 - Erros in CodeStripping.cs
 - line 128: StandanloneBuildSubtarget is not working in android

- maybe needs changed preprocessor
#if UNITY_EDITOR -> #if UNITY_EDITOR && !UNITY_ANDROID

2. Not working hosting on android device
- I don't know why this happening

**Comment by FirstGearGames** [comment]
2.7.10 doesn't exist.

**Comment by ymkim50** [comment]
Sorry 2.3.10.Pro

**Comment by Sleitnick** [comment]
Same/similar issue for my team. On v2.3.9.Pro. Full stack trace below:
```
ArgumentException: Subtarget property is not available for the platform (Android)
UnityEditor.Build.Reporting.BuildSummary.GetSubtarget[T] () (at <5f40cdb07bd44d76a23dad985a4ec283>:0)
FishNet.Configuring.CodeStripping.OnPreprocessBuild (UnityEditor.Build.Reporting.BuildReport report) (at Assets/Lib/FishNet/Runtime/Editor/CodeStripping.cs:128)
UnityEditor.Build.BuildPipelineInterfaces+<>c__DisplayClass16_0.<OnBuildPreProcess>b__1 (UnityEditor.Build.IPreprocessBuildWithReport bpp) (at <5f40cdb07bd44d76a23dad985a4ec283>:0)
UnityEditor.Build.BuildPipelineInterfaces.InvokeCallbackInterfacesPair[T1,T2] (System.Collections.Generic.List`1[T] oneInterfaces, System.Action`1[T] invocationOne, System.Collections.Generic.List`1[T] twoInterfaces, System.Action`1[T] invocationTwo, System.Boolean exitOnFailure) (at <5f40cdb07bd44d76a23dad985a4ec283>:0)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)
```

**Comment by FirstGearGames** [comment]
I'm looking into this. I cannot omit Android entirely on the script because that would result in code stripping being disabled for Android.

**Comment by FirstGearGames** [comment]
Fixed in 2.5.0
